### PR TITLE
add pfapack

### PR DESCRIPTION
**Pull request by basnijholt**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pfapack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c415dc8723ab9b5d96cda21b0fa115a36d6d1e758fc2c4943c2952b958dca81a
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/pfapack/meta.yaml]: This can be noarch python

Comment (basnijholt) [reply recipes/pfapack/meta.yaml]: Thanks! 

Is this https://github.com/conda-forge/staged-recipes/pull/10970/commits/b08bf9c679e860343132c84ab01b5b511e54baa0 sufficient?

Comment (basnijholt) [reply recipes/pfapack/meta.yaml]: Actually, I need `skip: True  # [py<36]` and this package doesn't work on Python 2.

Comment (scopatz) [reply recipes/pfapack/meta.yaml]: You can use `python >=3.6` in the requirements instead

Comment (basnijholt) [reply recipes/pfapack/meta.yaml]: Done! 👍

**Pull request merged**